swagger-v1/swagger.yaml

```diff
@@ -29,7 +29,7 @@ paths:
       description: |
         Retrieves the paginated list of a merchant's payment requests.
 
-        A valid **HMAC** is required to access this endpoint.
+        A valid **[MAC](https://docs.gocartpay.com/docs/api-authentication-hmac-3)** is required to access this endpoint.
```

Comment:
Is this URL going to change at some point since it also has "hmac" in it?

Reply:
good point... let me look into how to change that